bin/prepare

```diff
@@ -6,8 +6,7 @@ begin
   if ARGV[0]
     release_date = Date.parse ARGV[0]
   else
-    release_date = Date.today
```

Comment:
Maybe we can change to `Date.tomorrow` to prevent the Thursday cutting issue?

Reply:
Feature freeze is EOD on Thursday.  I think we should let that dictate the default.  If a few repos aren't ready, you can always override.  I would hope that the majority are ready on our actual cutoff.

Reply:
In actual practice, we've cut on Friday a few times. I think this should "just work" no matter if branches are cut on Thursday or Friday.

Reply:
Given that our cut off times can vary, I would recommend to always require the inclusion of the deploy date.

Reply:
Well, hopefully that becomes a rare occurrence and we stick with the established and agreed upon process.  I'll make the update though.